### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Logs Tab_Keyboard: Focus is lost after navigating from ‘Logs’ tab item

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3563]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS2.4.3; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
6) Tab till Next button and press enter. Press Finish button
7) In Project Explorer, Expand the created Scala project right-click src, New-->Other and press enter
8) Tab till Scala Wizards, expand and select 'Scala Object' and press next
9) Enter the name of the file and press Finish button
10) Open Scala file and write any code in the editor
11) In Project Explorer view, right-click your project, choose Azure-->Submit Spark Application to HDInsight
12) Enter all the required details and press submit button
13) In Azure Explorer tab, tab till HDInsight-->Cluster-->Job and press enter
14) Select any one the application and press enter. 
15) Tab till 'Logs' tab and press tab again
16) Verify if focus is lost after navigating form the Logs tab

**Actual Result:** 
Focus is lost after navigating from ‘Logs’ tab item

**Expected Result:** 
Focus should not be lost after navigating from the ‘Logs’ tab

**User Impact:** 
Users will find difficulty in navigating if focus is lost while navigating in the page

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_12_Focus lost after logs tab.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635627/Bug_12_Focus.lost.after.logs.tab.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `Logs`, press tab, focus is invisible, press tab again, focus is at `Open Spark History UI`.
![image](https://user-images.githubusercontent.com/32627233/65828091-a0777d00-e2c9-11e9-9391-1b7e3f858329.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by Tanushagottipati** [comment]
Closing this issue based on above comments

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749675
